05-services.md

```diff
@@ -1270,6 +1270,23 @@ networks:
 
 `mac_address` sets the MAC address used by the service container when connecting to this particular network.
 
+### driver_opts
+
+[![Compose v2.27.1](https://img.shields.io/badge/compose-v2.27.1-blue?style=flat-square)](https://github.com/docker/compose/releases/v2.27.1)
```

Comment:
I'd no idea which version to put here, so I made one up! But was aiming for "next" ... there's no need to rush this in.

Reply:
Please use 
```
[![unreleased](https://img.shields.io/badge/compose-unreleased-blue?style=flat-square)](https://github.com/docker/compose)
```
This help us track recent changes as docker compose is released and we update compose-spec accordingly

Reply:
Thank you! Done.